guides/taxation/value-added-tax.md

```diff
@@ -0,0 +1,107 @@
+# Value-added tax (VAT)
+
+Many countries have what is referred to as a value-added tax (VAT). In these
+cases, the tax is typically included in the price of each item. This means that
+no additional tax needs to be applied during checkout. However, many tax
+jurisdictions still require stores to show the amount of VAT that the customer
+paid.
+
+In the admin, the **Settings -> Taxes -> Tax Rates** page allows administrators
+to create any tax rate. They can create VAT-style taxes by using the "Included
+in price" checkbox.
+
+Solidus's `solidus_frontend` gem lists all of the VAT and other price
+adjustments below the item total on the checkout summary page.
+
+## Calculating VAT
+
+When you set up products in Solidus, you can set the price to the exactly what
+you want the customer to pay. Then, you can use your VAT-style tax rates to
+allocate a percentage of the gross price to taxes.
+
+```
+consumer_price / (1 + tax_rate) = expected_revenue
+consumer_price - expected_revenue = vat
+```
+
+Solidus's [Spree::Calculator::DefaultTax][default-tax-calculator] handles 
+sales tax and VAT. If a tax rate is VAT and should be included in the price, it 
+calculates all of the line items that share that tax rate on the order:
+
+```
+if rate.included_in_price
+  round_to_two_places(line_items_total - ( line_items_total / (1 + rate.amount) ) )
+...
+end
+```
+
+[default-tax-calculator]: https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/default_tax.rb
+
+### VAT amounts are stored in `Spree::Adjustment`s
+
+Note that while VAT does not adjust an order's total, Solidus still creates
+`Spree::Adjustment` objects to store tax amount. These objects have an 
+`included` value of `true` so that the tax is not added to the price.
+
+## Example order with multiple VAT rates
+
+In the following example, we will still refer to VAT as "adjustments",
```

Comment:
This sentence sounds like you shudder lightly at the implementation. :)

Reply:
I was just initially confused, since VAT would never "adjust" the price. Wanted to make sure it was clear for everyone!